modules/guides/pages/serverless/lambda.adoc

```diff
@@ -163,6 +163,6 @@ You can build and archive the function yourself with:
 
 [source,sh]
 ----
-go build github.com/benthosdev/benthos/v4/cmd/serverless/benthos-lambda
+go build github.com/redpanda-data/connect/v4/cmd/serverless/connect-lambda
 zip benthos-lambda.zip benthos-lambda
```

Comment:
Would this be `connect-lambda`?

Reply:
@JakeSCahill. I checked with Ash but also here is the repo path: [https://github.com/redpanda-data/connect/blob/main/cmd/serverless/connect-lambda/main.go](https://github.com/redpanda-data/connect/blob/main/cmd/serverless/connect-lambda/main.go)

Reply:
Checked and updated. Good spot!